### PR TITLE
Add template for the MNRAS (Monthly Notices of the Royal Astronomical Society) article.

### PR DESCRIPTION
**Pull request by oleskiewicz**
This creates a template for a submission to MNRAS journal, based on their LaTeX template file.  I've added most of the housekeeping as outlined [here](https://github.com/rstudio/rticles/issues/125#issuecomment-359955260), but wasn't sure what more is there to be done.

### CODE REVIEW

File: R/mnras_article.R [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+mnras_article <- function(...,
+                        keep_tex = TRUE,
+                        md_extensions = c(),
+                        fig_caption = TRUE){
+  pdf_document_format(...,
+                      keep_tex = keep_tex,
+                      md_extensions = md_extensions,
+                      format = "mnras_article",
+                      template = "template.tex",
+                      csl = "mnras.csl")
```
Comment (jjallaire) [inline R/mnras_article.R]: You need to use `system.file()` to refer to these (as the current code assumes they are in the current working directory)

**Comment by oleskiewicz** [comment]
Thanks @jjallaire, I'm just starting learning R so this is all a bit new to me!

I have changed the `template` and `csl` references using the `find_resource` helper function, just as `plos_article` [does it](https://github.com/rstudio/rticles/blob/371fee0e383daeb1b40d5914dfbdf434430dcb6a/R/plos_article.R#L24), I hope this is better now?

**Comment by jjallaire** [comment]
You should also run the tests locally (you can do this with `devtools::test()`). When I run your test I get this failure:

```r
unused argument (csl = "/Users/jjallaire/packages/rticles/inst/rmarkdown/templates/mnras_article/resources/mnras.csl")
1: rmarkdown::render(testdoc) at /Users/jjallaire/packages/rticles/tests/testthat/test_formats.R:30
2: create_output_format(output_format$name, output_format$options)
3: do.call(output_format_func, options)
```

I think you want to call `pdf_document_format()` rather than `pdf_document()`

**Comment by oleskiewicz** [comment]
Oh I see -- it seems like the LaTeX environment is trying to pick up the example figure (`example.png`), but it assumes the `.tex` extension and fails due to some relative path.

I'll change the way figure is rendered in the skeleton (it's my understanding that including figures through `knitr` is preferred anyway?), hopefully this will fix it.

Sorry it keeps failing, it built successfully for me.

**Comment by oleskiewicz** [comment]
The build fails for the same reason as previously:

    Package pdftex.def Error: File `example.png' not found: using draft setting.

I have checked other templates, and for instance `amq_article` passes the Travis build while including figures from external files in skeleton.  Similarly, there are no issues with the template locally.  How do I include the `example.png` in the test render?

File: inst/rmarkdown/templates/mnras_article/skeleton/skeleton.Rmd [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+worry about the exact layout, which will be handled by the publishers.
+
+Figures are referred to as e.g. Fig. \ref{fig:example_figure}, and tables as
+e.g. Table \ref{tab:example_table}.
+
+\begin{figure}
+  % To include a figure from a file named example.*
+  % Allowable file formats are eps or ps if compiling using latex
+  % or pdf, png, jpg if compiling using pdflatex
+  \includegraphics[width=\columnwidth]{example.png}
```
Comment (yihui) [inline inst/rmarkdown/templates/mnras_article/skeleton/skeleton.Rmd]: You included `example.png` here but the file is not included in the PR. You should either include a real file, or comment out this line.

**Comment by oleskiewicz** [comment]
The tests have passed!  Thank you for all your help -- let me know if there is anything else that needs fixing.

**Pull request merged**